pages/community/tsc.js

```diff
@@ -0,0 +1,246 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import TSCMembersList from "../../config/TSC_MEMBERS.json";
+
+export default function TSC() {
+  const description =
+    "See the current AsyncAPI TSC members and learn how you can become one.";
+  const image = "/img/social/parsers.png"; // TODO: change to a Screenshot of the page
+
+  function addAdditionalUserInfo(user) {
+    const userData = {
+      ...user,
+    };
+
+    addNameIfNotPresent(userData);
+    addSocialLinks(userData);
+    addAvatarUrl(userData);
+    userData.repos = userData.repos.map((repoName) =>
+      generateRepositoryInfo(repoName)
+    );
+
+    return userData;
+  }
+
+  function addNameIfNotPresent(user) {
+    // if username is not present, use the github username
+    if (!user.name) user.name = user.github;
+  }
+
+  function addSocialLinks(user) {
+    if (user.github) user.github = `https://www.github.com/${user.github}`;
+    if (user.linkedin)
+      user.linkedin = `https://www.linkedin.com/in/${user.linkedin}`;
+    if (user.twitter) user.twitter = `https://www.twitter.com/${user.twitter}`;
+  }
+
+  function addAvatarUrl(user) {
+    // github redirects to avatar url using `https://www.github.com/<username>.png`
+    user.avatarUrl = user.github + ".png";
+  }
+
+  function generateRepositoryInfo(repositoryName) {
+    return {
+      name: repositoryName,
+      url: `https://www.github.com/asyncapi/${repositoryName}`,
+    };
+  }
+
+  const tscMembers = TSCMembersList.map((user) => addAdditionalUserInfo(user));
+
+  return (
+    <GenericLayout
+      title="Technical Steering Committee"
+      description={description}
+      image={image}
+      wide
+    >
+      <div className="grid lg:grid-cols-3 lg:gap-8">
+        <div>
+          <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            What is TSC?
+          </h3>
+          <p className="my-4 text-base text-gray-500 lg:text-center">
+            TSC is something......
+          </p>
+        </div>
+        <div>
+          <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            Where is it coming from?
+          </h3>
+          <p className="my-4 text-base text-gray-500 lg:text-center">
+            Lorem ipsum dolor sit amet...
+          </p>
+        </div>
+        <div>
+          <h3 className="font-semibold  text-primary-800 mb-2 lg:text-2xl lg:text-center">
+            Governance model
+          </h3>
+          <p className="my-4 text-base text-gray-500 lg:text-center">
+            Lorem ipsum dolor sit amet...
```

Comment:
```suggestion
            AsyncAPI Initiative runs under an <a href="https://github.com/asyncapi/community/blob/master/CHARTER.md">an open governance model</a> that gives power to people actively involved and working on the project. No matter if you are an individual contributor or backed by a company, you have equal rights.  Read <a href="https://www.asyncapi.com/blog/governance-motivation">this</a> article in case you want to learn more.
```